model/schema/allele.yaml

```diff
@@ -225,15 +381,13 @@ classes:
           Can be null; if false, maybe you would show all the images.
           We need to revisit this issue.
 
-  AlleleNotesAssociation:
-    is_a: Association
-    description: >-
-      The relationship between an allele and a note.
-    slot_usage:
-      subject:
-        range: Allele
-      object:
-        range: Note
+  AlleleImageAssociationDTO:
+    is_a: AuditedObjectDTO
+    slots:
+      - allele_curie
+      - predicate_name
+      - image_curie
+      - primary_image
```

Comment:
because this is not inheriting from an 'evidence' slot-bearing class (like a hypothetical AssociationDTO class), 'evidence_curies' needs to be added here